src/file_name_plugin/__tests__/__snapshots__/plugin.test.ts.snap

```diff
@@ -1,127 +1,131 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`can select a pattern that matches multiple files 1`] = `
-
-[MOCK - cursorLeft]
-
- pattern › f
-[MOCK - cursorSavePosition]
-[MOCK - cursorLeft]
+"<eraseLine>
+<moveCursorToColumn1>
+<eraseScreenDown>
+<dim> pattern ›</> f
+<saveCursorPosition>
+<moveCursorToColumn1>
 
 
  Pattern matches 3 files
 
- › src/foo.js
-
- › src/file-1.js
-
- › src/file-2.js
-[MOCK - cursorTo(12, 5)]
-[MOCK - cursorRestorePosition]
+ <dim>›</> <dim>src/</></>f</><dim>oo.js</>
 
-[MOCK - cursorLeft]
+ <dim>›</> <dim>src/</></>f</><dim>ile-1.js</>
 
- pattern › fi
-[MOCK - cursorSavePosition]
-[MOCK - cursorLeft]
+ <dim>›</> <dim>src/</></>f</><dim>ile-2.js</>
+<moveCursorToRow6Column13>
+<restoreCursorPosition>
+<eraseLine>
+<moveCursorToColumn1>
+<eraseScreenDown>
+<dim> pattern ›</> fi
+<saveCursorPosition>
+<moveCursorToColumn1>
 
 
  Pattern matches 2 files
 
- › src/file-1.js
+ <dim>›</> <dim>src/</></>fi</><dim>le-1.js</>
 
- › src/file-2.js
-[MOCK - cursorTo(13, 5)]
-[MOCK - cursorRestorePosition]
+ <dim>›</> <dim>src/</></>fi</><dim>le-2.js</>
+<moveCursorToRow6Column14>
+<restoreCursorPosition>
+"
 `;
 
 exports[`can use arrows to select a specific file 1`] = `
-
-[MOCK - cursorLeft]
-
- pattern › f
-[MOCK - cursorSavePosition]
-[MOCK - cursorLeft]
+"<eraseLine>
+<moveCursorToColumn1>
+<eraseScreenDown>
+<dim> pattern ›</> f
+<saveCursorPosition>
+<moveCursorToColumn1>
 
 
  Pattern matches 3 files
 
- › src/foo.js
-
- › src/file-1.js
+ <dim>›</> <dim>src/</></>f</><dim>oo.js</>
 
- › src/file-2.js
-[MOCK - cursorTo(12, 5)]
-[MOCK - cursorRestorePosition]
+ <dim>›</> <dim>src/</></>f</><dim>ile-1.js</>
 
-[MOCK - cursorLeft]
-
- pattern › fi
-[MOCK - cursorSavePosition]
-[MOCK - cursorLeft]
+ <dim>›</> <dim>src/</></>f</><dim>ile-2.js</>
+<moveCursorToRow6Column13>
+<restoreCursorPosition>
+<eraseLine>
+<moveCursorToColumn1>
+<eraseScreenDown>
+<dim> pattern ›</> fi
+<saveCursorPosition>
+<moveCursorToColumn1>
 
 
  Pattern matches 2 files
 
- › src/file-1.js
-
- › src/file-2.js
-[MOCK - cursorTo(13, 5)]
-[MOCK - cursorRestorePosition]
+ <dim>›</> <dim>src/</></>fi</><dim>le-1.js</>
 
-[MOCK - cursorLeft]
-
- pattern › fi
-[MOCK - cursorSavePosition]
-[MOCK - cursorLeft]
+ <dim>›</> <dim>src/</></>fi</><dim>le-2.js</>
+<moveCursorToRow6Column14>
```

Comment:
In `ansi-escapes` position is zero based, but in vanilla escapes it is 1 base and also row argument goes first. That’s why `[MOCK - cursorTo(13, 5)]` becomes `<moveCursorToRow6Column14>`